LXProtocols.TCNet/Sockets/TCNetSocket.cs

```diff
@@ -467,7 +467,14 @@ protected void Broadcast(Socket socket, TCNetPacket packet)
             }
 
             packet.WriteData(writer);
-            socket.SendTo(data.ToArray(), new IPEndPoint(BroadcastAddress, localEndpoint.Port));
+            try
+            {
+                socket.SendTo(data.ToArray(), new IPEndPoint(BroadcastAddress, localEndpoint.Port));
+            }
+            catch (SocketException ex)
+            {
+                OnUnhandledException(new ApplicationException("Socket error broadcasting PioneerDJ integration.", ex));
```

Comment:
This should say TCNet as in theory the protocol allows for implementation with other manufacturers.